.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,34 @@
+name: Continuous Integration 
+
+on: [push]
```

Comment:
in this case you'll see an error right after you merge your code into master.
You need to see problems on stage, when you create MR.